runtime/manual/advanced/migrate_deprecations.md

```diff
@@ -12,11 +12,86 @@ will continue to be updated until the launch of 2.0.
 
 :::
 
-## API changes and deprecations
+## Subcommands
 
-The following APIs have changed or deprecated between Deno 1.x and 2.x, and will
-be removed in 2.x - guidance for migrating to a newer set of APIs is provided
-for each impacted API below.
+### deno bundle
+
+The `deno bundle` command has been removed. We recommend using
+[`esbuild`](https://esbuild.github.io/) together with
+[`esbuild-deno-loader`](https://jsr.io/@luca/esbuild-deno-loader).
+
+```ts
+import * as esbuild from "npm:esbuild";
+import { denoPlugins } from "jsr:@luca/esbuild-deno-loader";
+
+const result = await esbuild.build({
+  plugins: [...denoPlugins()],
+  entryPoints: ["https://deno.land/std@0.185.0/bytes/mod.ts"],
+  outfile: "./dist/bytes.esm.js",
+  bundle: true,
+  format: "esm",
+});
+
+esbuild.stop();
+```
+
+### deno vendor
+
+The `deno vendor` command has been replaced by a `"vendor": true` configuration
+option in `deno.json`.
+
+```json title="deno.json"
+{
+  "vendor": true
+}
+```
+
+## Flags
+
+### `--allow-none`
```

Comment:
```suggestion
### --allow-none
```

Reply:
AFAIK, symbols and parts of CLI commands are always expressed as inline code.

Reply:
Sorry, just saw your comment below. Ignore.